### ISSUE TITLE
Build failed. Uninstall "xxxxx" now?

### ISSUE THREAD

**Issue by mengqp** [wsdjeg/SpaceVim#51]
nvim installed 
run curl -sLf https://raw.githubusercontent.com/SpaceVim/SpaceVim/dev/install.sh | bash
when i open nvim and run call dein#install()
errors appear such as
![virtualbox_2017-01-06_13-54-34](https://cloud.githubusercontent.com/assets/8331712/21708897/b6b6e236-d417-11e6-99f5-6f94ba3ac8c7.jpg)

**Comment by wsdjeg** [comment]
I can not reproduc it, and `ctrlp-extensions.vim` has no `build` key. you maybe should try install plugin with VPN.

**Comment by wsdjeg** [comment]
`echo keys(dein#get()['ctrlp-extensions.vim'])` show me only:
`['lazy', 'name', 'repo', 'orig_opts', 'rtp', 'dummy_commands', 'merged', 'normalized_name', 'type', 'on_cmd', 'sourced', 'path']`

**Comment by wsdjeg** [comment]
`echo keys(dein#get()['vimproc.vim'])` will see `['lazy', 'repo', 'build', 'orig_opts', 'rtp', 'merged', 'normalized_name', 'type', 'name', 'sourced', 'path']`, this is plugin with `build` option.

**Comment by mengqp** [comment]
Perhaps I explained not very clear
I am using linuxmint18
every time"Build failed. Uninstall" xxxxx "now", I Select Yes
At the beginning, every time I can install several plug-ins, and finally can not install any plug-ins
Need to be noted that I can download normally, I can download and install other software such as spacemacs
I think the problem maybe  vpn, but it must be to have a vpn, I do not understand dein. Or a server for these plug-ins.

**Comment by mengqp** [comment]
I can use vpn successful installation of plugin 。

**Comment by wsdjeg** [comment]
what is your local.vim?

**Comment by wsdjeg** [comment]
and maybe you can run install script again to update SpaceVim.

**Comment by wsdjeg** [comment]
I can not reproduce this, clossing.

**Comment by AlphaLiu** [comment]
I also ran into this issue after I upgraded to the latest spacevim.
I am in MacOS, vim 8.0+

**Comment by wsdjeg** [comment]
But I can not reproduce, you can search build in this repo, only few plugins has this option.

**Comment by gbrlcustodio** [comment]
I also ran into this issue, almost every plugin installation shows this error.
I'm using Arch n nvim 0.1.7

**Comment by Shougo** [comment]
You should upload your vimrc and the reproduce ways.

**Comment by mandarvaze** [comment]
I'm also facing this issue for most (all?) the plugins (I installed SpaceVim today - Feb 16, 2017 if that helps) I'm selecting Yes when prompted.

Am on macOS using nvim 0.1.7

**Comment by mandarvaze** [comment]
I may have found a solution/root cause

I started seeing this for LOTs of the plugins. For me, the initial list was of 198 plugin.

Of which only (last?) 65 installed successfully
What caused the success (IMO) was some ``rmake`` (or was it ``rake``? Can't remember) worked successfully. Everything (?) **after that** installed successfully.

So I manually called ``call dein#install()`` once again

This time, I saw updated started for 133 plugins. It **all** went well.
✨  👏
